include/Renderer/Vertex.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <glm/vec2.hpp>
+#include <glm/vec3.hpp>
+
+struct Vertex
+{
+    glm::vec3 position;
+    glm::vec3 normal;
+    glm::vec2 uv;
```

Comment:
W plikach `.obj` te dwa parametry występują odwrotnie. Chcemy zostawić tak, czy trzymamy w kolejności jak w plikach?

Reply:
[Specyfikacja gltf 2.0 dotycząca geometrii](https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#geometry)
Jeżeli potem chcemy mieć kolejny format, to na razie i tak jest to bez znaczenia, można doprawdy to zmienić, ale nie widzę potrzeby jak u ciebie będzie to zmiana miejsca gdzie to wrzucasz, a po stronie renderera była by to zmiana bindowania atrybutu dla wierzchołka. Potem będzie pewnie bardziej flexible jak uda nam się zaimplementować layouty dla wierzchołków.